### PR TITLE
feat: token migration guides

### PR DESCRIPTION
**Pull request by caramis**

### CODE REVIEW

File: content/docs/token-migration-guides/about-token-migration.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Migration Period
+Oct. 08, 2022 ~ Jan. 05, 2023 (UTC)
+
+## Migration Web Application
+Migrate your `CTX` and other assets from [Migration Web Application](https://migration.c2x.world/).
+
+{{< hint info >}}
+**Note**  
+Please note that small transaction fees will be required when performing migration. If you do not have `USTC`, you can get `USTC` from Here where we are giving away `USTC` for free as a support measure. In order to receive the support, you must have more than 1 C2X and less than 0.2 `USTC` in your C2X Station.
```
Comment (jhlee-young) [inline content/docs/token-migration-guides/about-token-migration.md]: `Here` needs a link to `support.c2x.world`

Comment (caramis) [reply content/docs/token-migration-guides/about-token-migration.md]: Fixed

**Review by jhlee-young — APPROVED** [review]
LGTM

**Review by psy2848048 — APPROVED** [review]

File: content/docs/token-migration-guides/token-migration-procedure.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+## Migration Procedure
+
+{{< hint danger >}}
+**Important**  
+You must first agree to the following disclaimer by checking the box and clicking “Agree” before you proceed with the migration. Please keep in mind that in no event XPLA is liable for any loss that occurred during the migration process, so it is very important that you follow all of the given instructions to safely complete the migration procedure. Please refer to the whitepaper for detailed tokenomics behind XPLA chain.
+{{< /hint >}}
+
+{{< hint info >}}
+**Note**  
+The images below are still in development and are subject to change when it goes live
```
Comment (jhlee-young) [inline content/docs/token-migration-guides/token-migration-procedure.md]: needs period(.) at the end of this line

Comment (caramis) [reply content/docs/token-migration-guides/token-migration-procedure.md]: Fixed !

**Review by jhlee-young — APPROVED** [review]

**Pull request merged**